pypesto/objective/amici/amici.py

```diff
@@ -598,7 +598,7 @@ def set_custom_timepoints(
         amici_objective.apply_custom_timepoints()
         return amici_objective
 
-    def check_gradients_match_finite_differences(
+    def __check_gradients_match_finite_differences(
```

Comment:
```suggestion
    def _check_gradients_match_finite_differences(
```
Why not just a single underscore, marking this as a private function, instead of name mangling?

Reply:
Maybe I am missing something, but shouldn't gradient check be public?

Reply:
We have check_grad and check_grad_multi_eps already as per #723.

The method `check_garadients_match_finite_differences` I am using in a visualization routine in pypesto/visualize. (Will be added in a subsequent pull request, that's why method marked private as only used internally)